src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/LoadedProject.cs

```diff
@@ -72,7 +74,7 @@ public void Dispose()
         _projectSystemProject.RemoveFromWorkspace();
     }
 
-    public async ValueTask UpdateWithNewProjectInfoAsync(ProjectFileInfo newProjectInfo)
+    public async ValueTask<(ImmutableArray<CommandLineReference>, OutputKind)> UpdateWithNewProjectInfoAsync(ProjectFileInfo newProjectInfo)
```

Comment:
Might be worth a follow-up comment of why we're returning these very random things.